backend/src/main/java/co/inventorsoft/scripty/validation/PasswordConstraintValidator.java

```diff
@@ -10,13 +14,16 @@
  * @author Symyniuk
  *
  */
-public class PasswordConstraintValidator implements ConstraintValidator<ValidPassword, String> {
+public class PasswordConstraintValidator implements ConstraintValidator<ValidPassword, Password> {
     @Override
     public void initialize(ValidPassword constraintAnnotation) {
     }
 
     @Override
-    public boolean isValid(String password, ConstraintValidatorContext constraintValidatorContext) {
+    public boolean isValid(Password password, ConstraintValidatorContext constraintValidatorContext) {
+        if(password.getPassword() == null){
+            throw new ApplicationException("Please provide password", HttpStatus.BAD_REQUEST);
```

Comment:
Do not throw exception here. This method should signal if object is valid or not. Use `constraintValidatorContext` to put different errors for different cases